### DISCUSSION TITLE
FUNCTION OF FFI

### DISCUSSION THREAD

**Discussion by moyinmaala** [Cyfrin/foundry-full-course-cu#1438] [Q&A]
hi everyone, what is the function of FFI in foundry.toml files

**Comment by EngrPips** [comment]
Hey @moyinmaala  basically FFI enables you to run other programs from inside your test script, so if set to true in the foundry.toml files then you would have the luxury of running other programs from your test script. To just make things more clear enabling FFI in foundry.toml files simply means you would be able to run some Linux program or any other program directly from your test script. I hope this explanation help you understand the function of FFI in foundry.toml files
